test/integration/patients/worklist/worklist.js

```diff
@@ -3968,4 +3968,185 @@ context('worklist page', function() {
       .find('button')
       .should('not.exist');
   });
+
+  specify('actions with only work:team:manage permission', function() {
+    cy
+      .routeCurrentClinician(fx => {
+        fx.data.relationships.role = { data: { id: '77777' } };
+        fx.data.relationships.team = { data: { id: '11111', type: 'teams' } };
+
+        return fx;
+      })
+      .routeWorkspaceClinicians(fx => {
+        const teamMemberClinician = _.find(fx.data, { id: '22222' });
+        teamMemberClinician.attributes.name = 'Team Member';
+        teamMemberClinician.relationships.team.data.id = '11111';
+
+        const nonTeamMemberClinician = _.find(fx.data, { id: '33333' });
+        nonTeamMemberClinician.attributes.name = 'Non Team Member';
+        nonTeamMemberClinician.relationships.team.data.id = '22222';
+
+        return fx;
+      })
+      .routeActions(fx => {
+        fx.data = _.sample(fx.data, 4);
+
+        fx.data[0].attributes.name = 'Owned by current clinician’s team';
+        fx.data[0].attributes.created_at = testTsSubtract(1);
+        fx.data[0].relationships.state = { data: { id: '33333' } };
+        fx.data[0].relationships.owner = { data: { id: '11111', type: 'teams' } };
+
+        fx.data[1].attributes.name = 'Owned by another team';
+        fx.data[1].attributes.created_at = testTsSubtract(2);
+        fx.data[1].relationships.state = { data: { id: '33333' } };
+        fx.data[1].relationships.owner = { data: { id: '22222', type: 'teams' } };
+
+        fx.data[2].attributes.name = 'Owned by team member';
+        fx.data[2].attributes.created_at = testTsSubtract(3);
+        fx.data[2].relationships.state = { data: { id: '33333' } };
+        fx.data[2].relationships.owner = { data: { id: '22222', type: 'clinicians' } };
+
+        fx.data[3].attributes.name = 'Owned by non team member';
+        fx.data[3].attributes.created_at = testTsSubtract(4);
+        fx.data[3].relationships.state = { data: { id: '33333' } };
+        fx.data[3].relationships.owner = { data: { id: '33333', type: 'clinicians' } };
```

Comment:
These tests create a worklist for a user with the `work:team:manage` permission that includes 4 actions/flows in the list:

1. Owned by current clinician’s team (allowed to edit)
2. Owned by another team (not allowed to edit)
3. Owned by a fellow team member (allowed to edit)
4. Owned by a non-team member (not allowed to edit)

Then I test whether or not they are correctly editable vs. non-editable.